server/src/test/resources/server_test_data.sql

```diff
@@ -1,104 +0,0 @@
---
```

Comment:
I'm all for deleting unused files but we probably want to keep this one around -- we import this as sample data on keywhizdb_development for people to play with (https://square.github.io/keywhiz/#setup)